### PR TITLE
Yoga postures

### PR DESCRIPTION
**Pull request by zachthompson**
**What does your Instant Answer do?**

* Processes multiple source sites for yoga postures.
* Give individual postures, as well as a series of postures.

**What problem does your Instant Answer solve (Why is it better than organic links)?**

Provide a yoga posture, Sanskrit, and English translation without having to click.

**What is the data source for your Instant Answer? (Provide a link if possible)**

* http://www.ashtangayoga.info/
* http://www.yoga.com
* http://www.theyogaposes.com

**Why did you choose this data source?**

The first is a very reputable Ashtanga site, the second is very popular and well-visited, and the last attempts to be comprehensive.

**Are there any other alternative (better) data sources?**

Not that I know of.

**What are some example queries that trigger this Instant Answer?**

"downward dog posture", "boat pose", "marichyasana a", 'Nāḍi-Śodhana', 'ashtanga primary series', 'surya namaskara a'

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**

Yogis.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**

n/a, but has an [IA Page](https://duck.co/ia/view/yoga_asanas_api).

**Which existing Instant Answers will this one supercede/overlap with?**

n/a

**Are you having any problems? Do you need our help with anything?**

Yes, this IA data needs:

* Multiple sources
* Direct image display

**Where did you hear about DuckDuckHack? (For first time contributors)**

n/a

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

n/a

## Checklist
Please place an 'X' where appropriate.

```
[X] Added metadata and attribution information
[] Wrote test file and added to t/ directory
[] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
[] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
[] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 10

    - Windows 7
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 8
        [] IE 9
        [] IE 10

    - Windows XP
        [] IE 7
        [] IE 8

    - Mac OSX
        [] Google Chrome
        [] Firefox
        [] Opera
        [] Safari

    - iOS (iPhone)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - iOS (iPad)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - Android
        [] Firefox
        [] Native Browser
        [] Google Chrome
        [] Opera
```
https://duck.co/ia/view/yoga_postures

### CODE REVIEW

**Comment by b1ake** [comment]
@zachthompson Super cool! I've been hoping someone would do an asanas IA someday and that day has arrived!

File: share/longtail/yoga_postures/parse.pl [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+		
+	my ($title, $l2sm, $pp, $img, $meta) = @$d{qw(title l2sm pp img meta)};
+	$l2sm =~ s{/}{ }og;
+
+    print $output "\n", join("\n", 
+		qq{<doc>}, 
+		qq{<field name="title"><![CDATA[$title]]></field>},
+		qq{<field name="l2_sec_match2"><![CDATA[$l2sm]]></field>},
+		qq{<field name="paragraph"><![CDATA[$pp<br />[[Image:$img]]]]></field>},
+		qq{<field name="source"><![CDATA[YogaPosture]]></field>},
```
Comment (jdorweiler) [inline share/longtail/yoga_postures/parse.pl]: All lower case please.

Comment (jdorweiler) [inline share/longtail/yoga_postures/parse.pl]: Actually let's use yoga_posture as the ID to keep it consistent with the IA page.

Comment (zachthompson) [inline share/longtail/yoga_postures/parse.pl]: Should it be yoga_posture?

**Comment by zachthompson** [comment]
@b1ake I was going to cc you but you commented before I remembered.  You know of any other sources we should include?

File: share/longtail/yoga_postures/parse.pl (hunk 1 above)
Comment (zachthompson) [inline share/longtail/yoga_postures/parse.pl]: Ah, darn, don't worry about parse.pl.  That's getting nuked.  process.pl has it's logic.

**Comment by jdorweiler** [comment]
Needs a readme file too with a list of any modules needed and how to install.

**Comment by b1ake** [comment]
@zachthompson As far as sources, I think you found a bunch of good ones.

The only source I can maybe add to that is this Wikipedia page, which links to articles on many asanas, including pictures and descriptions:

https://en.m.wikipedia.org/wiki/List_of_asanas

Not sure if it's additive and useful but figured I'd throw it out there. 

I really wish we could use the images and descriptions from the book Light on Yoga, as they are awesome. Don't thing we can though for copyright reasons, and also because I don't see them digitized in a clean way (although didn't look that hard). 

Btw I think it's really neat that you have full ashtanga sequences. :)

**Comment by zachthompson** [comment]
@jdorweiler I think we're ready to roll when you get back.

**Comment by jdorweiler** [comment]
:godmode:

**Comment by zekiel** [comment]
doesn't appear this was tested on Safari bc images are broken in Safari (tested on mac) 
yoga

**Comment by zachthompson** [comment]
[The spice](https://github.com/duckduckgo/zeroclickinfo-spice/pull/1964)

**Pull request merged**